indico/modules/events/contributions/forms.py

```diff
@@ -69,6 +69,10 @@ def __init__(self, *args, submitter_edit=False, **kwargs):
         self.type.query = self.event.contribution_types
         if self.event.type != 'conference':
             self.person_link_data.label.text = _('Speakers')
+        if self.contrib and self.contrib.abstract:
+            self.person_link_data.default_is_author = True
```

Comment:
Will this break in the - albeit rare and obscure - case of a conference event where you created a contribution from an abstract, and then changed the event type to meeting, where you no longer have authors?

Reply:
it shouldn't, i'm always creating the `default_is_author` attr in the field's init regardless if we have authors

Reply:
just tested it, it works totally fine except it keeps using the abstract defaults since the abstract is still there (but hidden).
I guess i can fix that just by changing that `if` to an `elif` 😄